be/src/env/cloud_storage_client.h

```diff
@@ -0,0 +1,53 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#pragma once
+
+#include <string>
+
+#include "common/status.h"
+
+namespace starrocks {
+
+// base class for cloud storage client
+class CloudStorageClient {
```

Comment:
Could you please move this class into another directory, such as be/src/object_sotre?